peekingduck/pipeline/nodes/preprocess/contrast.py

```diff
@@ -40,22 +36,26 @@ class Node(AbstractNode):
         |img|
 
     Configs:
-        brightness (:obj:`int`): **[-100,100], default = 0**. |br|
-            Adjusts the brightness of the image.
-        contrast (:obj:`float`): **[1, 3], default = 1**. |br|
-            Adjusts the contrast of the image.
+        alpha (:obj:`float`): **[1, 3], default = 1**. |br|
```

Comment:
I think with this range, the user can only increase contrast but not decrease, is there a reason why we don't want they to be able to decrease the contrast?

Reply:
Most of the opencv guides I saw use this range of between 1 to 3, but you are right, there may be a scenario where the lighting is harsh in comparison with the foreground object, and contrast should be lowered. Updated to allow alpha to be between 0 and 1 as well.